bundles/action/org.openhab.action.ciscospark/README.md

```diff
@@ -0,0 +1,38 @@
+# Cisco Spark Action
+
+
+Documentation of the Cisco Spark Action
+
+## Introduction
+
+[Cisco Spark](https://www.ciscospark.com) is a simple and secure way for collaborating with individuals and groups across the cloud.  This Action plugin allows you to send messages to _rooms_ and individuals when certain events take place in openHAB. Messages in the Cisco Spark plugin support Markdown text.
```

Comment:
please remove lines 3-7 inclusive.  This file is processed to appear on docs.openhab.org so a particular set of conventions are expected.